spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerTemplate.java

```diff
@@ -539,12 +540,17 @@ private void resolveChildEntity(Object entity, Set<String> includeProperties) {
 					Class childType = spannerPersistentProperty.getColumnInnerType();
 					SpannerPersistentEntity childPersistentEntity = this.mappingContext
 							.getPersistentEntity(childType);
+
+					Supplier<List> childPropertyFunc = () -> queryAndResolveChildren(childType,
```

Comment:
Maybe rename var to something like `getChildrenRowsFunc`? (Currently not 100% sure what it is doing).

Reply:
done!